yarn.lock

```diff
@@ -1216,7 +1216,7 @@
     iterall "^1.1.3"
     uuid "^3.1.0"
 
-"@lesswrong/lesswrong-editor@./public/lesswrong-editor/":
+"@lesswrong/lesswrong-editor@./public/lesswrong-editor":
```

Comment:
I'd really look into `yarn` workspaces for this use case, then you don't have to recompile and re-import this every time it changes, it'll just work. Essentially, `yarn` is providing a symlink from any workspace folder with a package. (Note that you need to do a `yarn add` step to add the `lesswrong-editor` package to the `lesswrong` package, and you need to specify the version number, otherwise `yarn` tries to lookup the package name on NPM)

Reply:
Obvs not something for this PR, just something to think about that might simplify things a lot once it's set up.

Reply:
Yeah, I came across workspaces a few weeks ago actually, and was like, "next time I have a problem with this I'll try the refactor"